java/find-place/src/main/java/com/esri/arcgisruntime/sample/findplace/MainActivity.java

```diff
@@ -141,7 +142,7 @@ protected void onCreate(Bundle savedInstanceState) {
     mProximitySearchViewEmpty = true;
 
     // create a LocatorTask from an online service
-    mLocatorTask = new LocatorTask(getString(R.string.world_geocode_service));
+    mLocatorTask = new LocatorTask("https://geocode-api.arcgis.com/arcgis/rest/services/World/GeocodeServer");
```

Comment:
Thanks @Rachael-E - It's an old pattern where I was too trusting of Android's linter. It'd regularly complain about strings in code rather than in strings.xml. This is for localization purposes though--so a translator could translate words from one app version to another.

Since that's non-sensical for URLs, we're moving towards URLs being in code and plain english in strings.xml.

Thanks for checking though!